integration-tests/testconfig/README.md

```diff
@@ -185,168 +170,212 @@ To set custom per-chain config use `[NodeConfig.ChainConfigTOMLByChainID]`. Exam
+use = false # Default: true. Reuse existing OCR contracts?
+configure = false # Default: true. Configure existing OCR contracts?
 
-## To bear in mind
+- - - - - - - - - - - - 
 
-### Validation failures
+# Non-compliant configurations
+[OCR.Contracts]
+link_token = "0x88d1239894D9582f5849E5b5a964da9e5730f1E6"
+offchain_aggregators = ["0xc1ce3815d6e7f3705265c2577F1342344752A5Eb", "0x2f4FA21fCd917C448C160caafEC874032F404c08"]
 
-When the system encounters even a single setting related to a specific product or configuration within the configurations, it triggers a comprehensive validation of the entire configuration for that product. This approach is based on the assumption that if any configuration for a given product is specified, the entire set of configurations for that product must be complete and valid. This is particularly crucial when dealing with the `overrides.toml` file, where it's easy to overlook the need to comment out or adjust values when switching between configurations for different products. Essentially, the presence of any specific configuration detail necessitates that all relevant configurations for that product be fully defined and correct to prevent validation errors.
+# Example 1: Setting `configure` to `false` for selected (not all) addresses will fail configuration validation
+[OCR.Contracts.Settings."0xc1ce3815d6e7f3705265c2577F1342344752A5Eb"]
+configure = false
 
-## Possible nil pointers
+# OR
 
-If no configuration values are set for a product or its logging parameters, the system won't perform validation checks. This can lead to a 'nil pointer exception' error if you attempt to access a configuration property later on. This situation arises because we use pointers to facilitate optional overrides; accessing an unset (nil) pointer will cause an error. To avoid such issues, especially when general validations might not cover every scenario, it's crucial for users to ensure that all necessary configuration options are explicitly set. Additionally, it's highly recommended to implement test-specific validations to confirm that all required values for a particular test are indeed established. This proactive approach helps prevent runtime errors and ensures smooth test execution.
+# Example 2: Setting `configure` to different values for the listed contracts will fail execution
+[OCR.Contracts.Settings."0xc1ce3815d6e7f3705265c2577F1342344752A5Eb"]
+configure = false
 
-## Contributing
+[OCR.Contracts.Settings."0x2f4FA21fCd917C448C160caafEC874032F404c08"]
+configure = true
 
-It's crucial to incorporate all new test configuration settings directly into the TOML configuration files, steering clear of using environment variables for this purpose. Our goal is to centralize all configuration details, including examples, within the same package. This approach simplifies the process of understanding the available configuration options and identifying the appropriate values to use for each setting.
```

Comment:
IMO I'd leave this to underline our strategy here

Reply:
Moved it to the very top of the doc (under [!IMPROTANT] alert), thanks